templates/NewDetector_root_containers/NewDetectorGeo.h

```diff
@@ -1,37 +1,25 @@
 /********************************************************************************
- * Copyright (C) 2014-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ * Copyright (C) 2014-2024 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
  *                                                                              *
  *              This software is distributed under the terms of the             *
  *              GNU Lesser General Public Licence (LGPL) version 3,             *
  *                  copied verbatim in the file "LICENSE"                       *
  ********************************************************************************/
+
 #ifndef NEWDETECTORGEO_H
 #define NEWDETECTORGEO_H
 
 #include "FairGeoSet.h"
 
 class NewDetectorGeo : public FairGeoSet
 {
-  protected:
-    char modName[20]{"NewDetector"};   // name of module
-    char eleName[20]{"NewDetector"};   // substring for elements in module
   public:
```

Comment:
If everything is public anyway, maybe use a struct?

Reply:
I guess, this should be discussed in general, how people feel about using `struct` here.

In any case, it is out of scope of this PR imho.